### PR TITLE
awesome books es6, linters, luxon

### PR DESCRIPTION
**Pull request by King-Benx**
**Description:**
This delivers the awesome book project with es6 modules.

**Extra project requirements**
- Time functionality with luxon

**How to run**
1. Clone the project ```git clone git@github.com:King-Benx/awesome-books-es6.git```
2. Checkout to the root directory.
3. Install dependencies ```npm i```
4. Right click your index file and open with live server

**Screenshots**
Screenshot 2023-02-21 at 12-20-59 Awesome Books
Screenshot 2023-02-21 at 12-22-05 Awesome Books
Screenshot 2023-02-21 at 12-22-15 Awesome Books

### CODE REVIEW

File: modules/Book.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      ...allBooks,
+      { id: Math.floor(Math.random() * 100) + 1, title, author },
+    ];
+    this.updateStorage(updatedData);
+  }
+
+  removeBook(id) {
+    const allBooks = this.listOfBooks;
+    const updatedData = allBooks.filter((book) => book.id.toString() !== id);
+    this.updateStorage(updatedData);
```
Comment (brainconnect93) [inline modules/Book.js:37]: - [ ] Great effort working on the awesome book project 👍 , but currently, in your project, I noticed I could not add or remove books, anytime a user tries to add a book using the form in the `Add New` section it's not adding and in the `List` section it's not displaying any added book which makes it difficult to remove a book from the list. Kindly fix this.
- [ ] Also in the `List` section you should not have a form asking us to add a book. the `List` is only meant to show the books we have added from the `Add New` section and also the `List` section gives room to remove any book added. Kindly remove the form.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @King-Benx, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/Book.js (hunk 1 above)
Comment (King-Benx) [reply modules/Book.js:37]: Clear your localstorage and try again

**Review by emmiiorji — APPROVED** [review]
Hi @King-Benx, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**